### ISSUE TITLE
Platypus: Home link missing discernible text

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#2070]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page's home link (icon at the top left) is missing a discernible text.

---

https://dequeuniversity.com/rules/axe/4.3/link-name?application=axeAPI

---

**User impact:** Serious
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** Section 508: MUST
**WCAG Success Criteria:** 2.4.4 Link Purpose (In Context), 4.1.2 Name, Role, Value
